testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1201,6 +1201,38 @@
   end
 end
 
+Then(/^"([^"]*)" virtual machine on "([^"]*)" should boot using autoyast$/) do |vm, host|
+  node = get_target(host)
+  output, _code = node.run("virsh dumpxml #{vm}")
+  tree = Nokogiri::XML(output)
+  has_kernel = tree.xpath("//os/kernel").size == 1
+  has_initrd = tree.xpath("//os/initrd").size == 1
+  has_autoyast = tree.xpath("//os/cmdline")[0].to_s.include? ' autoyast='
+  raise 'Wrong kernel/initrd/cmdline configuration' unless has_kernel && \
```

Comment:
Well... as you have the info already... why not offer the valures of those booleans into the raised message?

Reply:
good point